airflow/example_dags/tutorial.py

```diff
@@ -97,7 +97,7 @@
     """
     )
 
-    dag.doc_md = __doc__  # providing that you have a docstring at the beginning of the DAG
+    dag.doc_md = __doc__  # providing that you have a docstring at the beginning of the DAG; OR
```

Comment:
```suggestion
    dag.doc_md = __doc__  # providing that you have a docstring at the beginning of the DAG
```

accidental?

Reply:
No, intentional but probably shouldn't be in this PR. (The tutorial is showing multiple ways of setting `dag.doc` -- you wouldn't want to do all of them.)

Reply:
Oh, this needs to be read with a next lines as one sentence. Got it now.